conn_test.go

```diff
@@ -79,33 +79,4 @@ func TestDeadlockInClose(t *testing.T) {
 	case <-time.After(3 * time.Second):
 		t.Fatal("apparent deadlock!")
 	}
```

Comment:
Why did you remove the test?

Reply:
At that time, I wrote a unit test for `GetEphemerals`, which uses the ZK service of direct connection 127.0.0.1. This requires me to start a ZK service locally. However, after the first two commits were submitted, GitHub's default unit test check failed directly. So I tried to remove this unit test and verify whether the automatic check would pass. Obviously, another problem I mentioned above appeared One question (How to fix the 'Codecov Report'?) Maybe I should comment on the test? Because it seems that there is no corresponding test method for other public methods